deathcause.lua

```diff
@@ -12,25 +13,24 @@ function getItemAtPosition(pos)
     end
 end
 
-function getRaceNameSingular(race_id)
+local function getRaceNameSingular(race_id)
     return df.creature_raw.find(race_id).name[0]
 end
 
-function getDeathStringFromCause(cause)
+local function getDeathStringFromCause(cause)
     if cause == -1 then
         return "died"
     else
         return DEATH_TYPES[cause]:trim()
     end
 end
 
-function displayDeathUnit(unit)
+function getDeathUnit(unit)
```

Comment:
This is now an API function, so it needs a better name. "printDeathUnit" may have made enough sense when this was just a script, but `reqscript('deathcause').getDeathUnit()` is not a name that makes the behavior of the function or the return value obvious.

Try to think of this from the caller's perspective? What do they need to know about what this function does, and how can you communicate that in the name of the function?

This comment goes for all the other non-`local` functions below.